multiple-domain/multiple-domain.php

```diff
@@ -46,6 +46,12 @@
 $domainLang = $multipleDomain->getDomainLang();
 
 
+/**
```

Comment:
There is a method called `initFilters` inside the MultipleDomain class. Could you move this `add_filter` call there, please?

Reply:
I haven't noticed that method, thanks for pointing it out! I moved the `add_filter` call and updated my PR.

Thanks for the quick reply, btw 😊